packages/core-components/api-report.md

```diff
@@ -460,6 +470,15 @@ export function HeaderLabel(props: HeaderLabelProps): JSX.Element;
 // @public (undocumented)
 export type HeaderLabelClassKey = 'root' | 'label' | 'value';
 
+// @public (undocumented)
+export type HeaderMenuActionItem = {
```

Comment:
Realized this should obviously be `HeaderActionMenuItem` instead 😅 

Or `HeaderActionMenuActionItem`, but perhaps we don't need to go quite that far. Then again if we don't, perhaps the prop should be `items` rather than `actionItems`? I don't feel these are particularly action-y anyway, they're just menu items right?